concordium-consensus/src/Concordium/GlobalState/Persistent/BlobStore.hs

```diff
@@ -347,8 +347,8 @@ readBlobBS bs@BlobStoreAccess{..} br@(BlobRef offset) = do
 
 -- |Check if a given 'BlobRef' is valid in the blob store. This does not attempt to deserialize
```

Comment:
I guess, this comment is not accurate after having changed `readBlobBS` to `loadRef`. The former does not deserialize the blob, but the latter does.

Reply:
I fixed the comment.